aiven/resource_service_integration.go

```diff
@@ -279,9 +227,8 @@ func resourceServiceIntegrationRead(_ context.Context, d *schema.ResourceData, m
 		return nil
 	}
```

Comment:
It seems the code above is more indented

Reply:
Looks like it's something on GitHub side

Reply:
Indentation looking weird could be a tabs/spaces issue.

Reply:
i double checked; indentation is fine here; seems to be a github thing